### PR TITLE
Remove EMA hook for classification task

### PR DESCRIPTION
**Pull request by supersoob**
This PR includes

- Remove EMA hook for classification task
- Lower priority of EvalHook

### CODE REVIEW

**Review by jaegukhyun — CHANGES_REQUESTED** [review]
There are some of flake8 issues

**Review by goodsong81 — APPROVED** [review]
LGTM. But not sure to remove it also from Semi-SL recipe.
@harimkang Could you confirm?

**Review by JihwanEom — APPROVED** [review]
LGTM

**Review by jaegukhyun — APPROVED** [review]
LGTM

File: recipes/stages/classification/semisl.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ optimizer_config:
 
 custom_hooks:
 - type: NoBiasDecayHook
-- type: ModelEmaV2Hook
-- type: SemiSLClsHook
```
Comment (harimkang) [inline recipes/stages/classification/semisl.yaml:20]: I think SemiSL should first leave the EMA as it is.

**Review by harimkang — CHANGES_REQUESTED** [review]

**Review by harimkang — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**